tests/ha/fencing.pm

```diff
@@ -18,43 +18,56 @@ use lockapi;
 use hacluster;
 
 sub run {
-    my $cluster_name = get_cluster_name;
+    my $cluster_name  = get_cluster_name;
+    my $node_to_fence = get_var('NODE_TO_FENCE', undef);
+    my $node_index    = !defined $node_to_fence ? 1 : 2;
 
     # Check cluster state *before* fencing
-    barrier_wait("CHECK_BEFORE_FENCING_BEGIN_$cluster_name");
+    barrier_wait("CHECK_BEFORE_FENCING_BEGIN_${cluster_name}_NODE${node_index}");
     check_cluster_state;
-    barrier_wait("CHECK_BEFORE_FENCING_END_$cluster_name");
+    barrier_wait("CHECK_BEFORE_FENCING_END_${cluster_name}_NODE${node_index}");
 
     # Give time for HANA to replicate the database
-    #    if (check_var('AUTOMATED_REGISTER', 'true')) {
     if (check_var('CLUSTER_NAME', 'hana')) {
-        # TODO: Fix
-        sleep 300;
-        assert_script_run "SAPHanaSR-showAttr";
+        sles4sap->check_replication_state;
```

Comment:
Same comment as above with regards to calling class methods without instancing an object.